src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -18,9 +19,37 @@ import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import type {Transaction} from '@src/types/onyx';
 import type {SearchPersonalDetails, SearchPolicyDetails, SearchTransactionType} from '@src/types/onyx/SearchResults';
+import type IconAsset from '@src/types/utils/IconAsset';
 import BaseListItem from './BaseListItem';
 import type {ListItem, TransactionListItemProps, TransactionListItemType} from './types';
 
+type TextWithIconCellProps = {
+    icon: IconAsset;
+    text: string;
+    showTooltip: boolean;
+};
+
+function TextWithIconCell({icon, text, showTooltip}: TextWithIconCellProps) {
```

Comment:
Perhaps we can move this small component to a separate file? I wouldn't move it outside to `components` because nothing yet uses it. But maybe a file in here right next to `TransactionListItem.tsx`?

Im not sure what are the practices in this codebase, but I think it would make sense.

Reply:
Yea, I agree we should move this to a separate file in SelectionList/